webapp/app/forms/steps/eligibility_steps.py

```diff
@@ -471,9 +471,10 @@ class InputForm(SteuerlotseBaseForm):
             label="",
             render_kw={'hide_label': True,
                        'data-detail': {'title': _l('form.eligibility.elster_abrufcode_eligibility.detail.title'),
-                                  'text': _l('form.eligibility.elster_abrufcode_eligibility.detail.text')}},
+                                       'text': _l('form.eligibility.elster_abrufcode_eligibility.detail.text')}},
             choices=[('yes', _l('form.eligibility.elster_abrufcode_eligibility.yes')),
                      ('no', _l('form.eligibility.elster_abrufcode_eligibility.no')),
+                     ('none', _l('form.eligibility.elster_abrufcode_eligibility.none')),
```

Comment:
Isn't `'none'` potentially quite confusing because it's so similar to Python's `None`? How about `'maybe'` or `'unknown'`?